shuup/admin/templates/shuup/admin/base/_navigation.jinja

```diff
@@ -25,6 +25,7 @@
     <a class="nav-btn shop-btn" href="{{ url("shuup_admin:home") }}" data-toggle="tooltip" data-placement="bottom" title="{% trans %}Go home{% endtrans %}"><i class="fa fa-home"></i></a>
 
     {% set front_url=shuup_admin.get_front_url() %}
+    {% if front_url == "/None" %}{% set front_url="/" %}{% endif %}
```

Comment:
what is this?

Reply:
This is due to that I was getting a url for "/None" from `shuup_admin.get_front_url()`, then because "/None" is not False in the if  that leads you to get a url to "/None" and that leads to a 404 page.